src/shared/components/Typography/styles.ts

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+import { StyledTypographyProps } from './types';
+import { css } from '@emotion/react';
+
+export const H1 = styled.h1<StyledTypographyProps>`
+  font-size: 40px;
+  font-weight: 600;
+  margin: 0;
```

Comment:
**Suggestion:** I see a lot repetition for `margin: 0`. How about moving it to `src/index.css` file. 

```css
h1,
h2,
h3,
p {
  margin: 0;
}
```